common/lib/xmodule/xmodule/capa_module.py

```diff
@@ -2244,6 +2244,14 @@ def score_from_lcp(self, lcp):
         return Score(raw_earned=lcp_score['score'], raw_possible=lcp_score['total'])
 
 
+class ProblemBlock(_ProblemBlock):
+    """
+    @medality_custom
+    This is only to test that problem block passes unit tests when extended
+    """
+    pass
```

Comment:
this might be overkill, but it allows us to test that unit tests pass with a subclass of ProblemBlock.